dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/database/DatabaseInfo.java

```diff
@@ -140,6 +144,19 @@ public void setSpatialSupport( boolean spatialSupport )
         this.spatialSupport = spatialSupport;
     }
 
+    @Override
+    public DatabaseInfo clone()

```

Comment:
We actually don't use clone in the system due to flaws with the clone interface. Could we refactor to use `BeanUtils.copyProperties` instead?

Ref: https://books.google.com/books?id=ka2VUBqHiWkC&pg=PA55&lpg=PA55&dq=effective+java+clone&source=bl&ots=yXGhLnv4O4&sig=zvEip5tp5KGgwqO1sCWgtGyJ1Ns&hl=en&ei=CYANSqygK8jktgfM-JGcCA&sa=X&oi=book_result&ct=result&resnum=3#v=onepage&q=effective%20java%20clone&f=false

Reply:
Has been replaced.